src/web/mjs/connectors/Mangafest.mjs

```diff
@@ -0,0 +1,94 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Mangafest extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangafest';
+        super.label = 'Mangafest X';
+        this.tags = [ 'indonesian', 'webtoon'];
+        this.url = 'https://mangafest.net';
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/category/manga/', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'nav.navigation.pagination div.nav-links a.page-numbers:not(.next)');
+        const pages = parseInt(data[data.length-1].pathname.match(/\d+\/$/));
+
+        let mangas = [];
+        for (let page = 1; page <= pages; page++) {
+            request = new Request(new URL('/category/manga/page/'+page+'/', this.url), this.requestOptions);
+            data = await this.fetchDOM(request, 'div.post_image a');
+            mangas.push(...data.map(manga => {
+                return {
+                    id: this.getRootRelativeOrAbsoluteLink(manga.pathname, request.url),
```

Comment:
May be safer to use `this.url` instead of `request.url`, in case both are different (cross reference), then `getPages` would not work as it assumes `this.url` as base

Reply:
ahh careless mistake